src/containers/ColorByMenu/selectors.ts

```diff
@@ -139,11 +143,22 @@ export const getInteractivePanelData = createSelector(
             {} as { [key: string]: number }
         );
 
+        if (countsByCategory[MISSING_CATEGORY_KEY] > 0) {
+            // Some data points have no data for this category. Modify categories
+            // to include an N/A entry.
+            categories = [
```

Comment:
Nit: I wonder if this would be a little more efficient to just do `categories.push({color,name,key})` instead of the full spread operator